soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1191,81 +1191,537 @@ s16 Randomizer::GetRandomizedItemId(GetItemID ogId, s16 actorId, s16 actorParams
 }
 
 s16 Randomizer::GetItemFromActor(s16 actorId, s16 actorParams, s16 sceneNum, GetItemID ogItemId) {
-    return GetItemFromGet(this->itemLocations[GetCheckFromActor(sceneNum, actorId, actorParams)], ogItemId);
+    return GetItemIDFromRandomizerGet(this->itemLocations[GetCheckFromActor(sceneNum, actorId, actorParams)], ogItemId);
```

Comment:
might be cool to change the name of this to `GetItemIdFromActor`

Reply:
I might would actually go the other way and pull the IsVanillaItem and ItemTable_RetrieveEntry logic into here to make the name fit, but maybe that deserves it's own PR.

Reply:
I renamed a few of these in #1323 but since that probably wont be merged for a while I can pull some of those changes over here instead

Reply:
Actually I take that back, gonna leave this for a separate PR